app/views/public/api_documentation.html.erb

```diff
@@ -364,9 +365,29 @@
     <p>Parameters:</p>
     <p><strong>`time: NN`</strong> Confirmation time in miliseconds</p>
     <p><strong>`signature: string`</strong> Transaction signature</p>
+    <p><strong>`reported_at: %year-%month-%dayT%hour:%minute:%second`</strong> Transaction creation time</p>
     <p><strong>`transaction_type: string`</strong> A string to describe the type of transaction. e.g. 'transfer', 'defi-swap', 'nft-mint', or anything meaningful to the sender.</p>
     <p>Response:</p>
-    <pre>{"status":"ok"}</pre>
+    <pre>{"status":"created"}</pre>
+  </div>
+</div>
+
+<hr />
+
+<h4 id="ping-thing-batch">Ping Thing Batch</h4>
+<div class="row">
+  <div class="col-sm-12">
+    <p>The ping thing batch endpoint allows you to push multiple transaction informations at once.</p>
+    <p>Request:</p>
+    <pre>curl -X POST -H "Token: secret-api-token" -H "Content-Type: application/json" -d '{"transactions": ["time": "NN", "signature": "signature", "transaction_type": "transfer"}]'
```

Comment:
`reported_at` is missing, please add it also above. I think we should stick to ISO 8601 format as it is a general consensus for sending datetimes in json https://stackoverflow.com/a/15952652/3868595

Also I think you missed  curly brace  before "time" and at the end of the object inside array. There is also wrong order at the end. It would be nice to provide at least two objects in the example array since it's a batch.